docs/gguf.md

```diff
@@ -34,6 +34,9 @@ The components are:
     - `M`: Million parameters.
     - `K`: Thousand parameters.
 5. **EncodingScheme**: Indicates the weights encoding scheme that was applied to the model. Content, type mixture and arrangement however are determined by user code and can vary depending on project needs.
+6. **Shard**: (Optional) Indicates and denotes that the model has been split into multiple shards, formatted as `<ShardNum>-of-<ShardTotal>`.
+    - *ShardNum* : Shard position in this model. Must be at least 5 digits padded by zeros.
+    - *ShardTotal* : Total number of shards in this model. Must be at least 5 digits padded by zeros.
```

Comment:
```suggestion
    - *ShardTotal* : Total number of shards in this model. Must be 5 digits padded by zeros.
```

we can remove the `at least` part, right?

Reply:
done